Explorer/Assets/Scripts/ECS/Unity/Transforms/Systems/SyncGlobalTransformSystem.cs

```diff
@@ -0,0 +1,67 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using DCL.Character;
+using DCL.Character.Components;
+using DCL.CharacterCamera;
+using DCL.SDKComponents.TransformSync.Components;
+using DCL.Utilities;
+using ECS.Abstract;
+using ECS.Groups;
+using ECS.Unity.Transforms.Components;
+using UnityEngine;
+
+namespace ECS.Unity.Transforms.Systems
+{
+    /// <summary>
+    ///     This system syncs the Camera and Player transforms to specially created entities in each SDK scene
+    /// </summary>
+    [UpdateInGroup(typeof(SyncedPreRenderingSystemGroup))]
+    public partial class SyncGlobalTransformSystem : BaseUnityLoopSystem
+    {
+        private readonly ObjectProxy<World> globalWorldProxy;
+        private SingleInstanceEntity cameraEntityProxy;
+        private SingleInstanceEntity playerEntityProxy;
+
+        private SyncGlobalTransformSystem(World world, ObjectProxy<World> globalWorldProxy) : base(world)
+        {
+            this.globalWorldProxy = globalWorldProxy;
+        }
+
+        public override void Initialize()
+        {
+            if (!globalWorldProxy.Configured)
+                return;
+
+            World globalWorld = globalWorldProxy.Object!;
+            cameraEntityProxy = globalWorld.CacheCamera();
+            playerEntityProxy = globalWorld.CachePlayer();
+        }
+
+        protected override void Update(float t)
+        {
+            UpdateCameraTransformQuery(World);
+            UpdatePlayerTransformQuery(World);
+        }
+
+        [Query]
+        [All(typeof(CameraTransformSync))]
+        private void UpdateCameraTransform(ref TransformComponent transformComponent)
+        {
+            if (!globalWorldProxy.Configured) return;
```

Comment:
Is this possible?

Reply:
Sadly, yes, with the Cube scene unit test haha